.vscode/settings.json

```diff
@@ -26,6 +26,10 @@
 		}
 	],
 	"npm.packageManager": "yarn",
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
out .eslint has prettier built in (https://github.com/ParabolInc/parabol/blob/1b2eb07d7804280847c616bdc372b7c61c83f0cb/.eslintrc.js#L3)
I wonder it hasn't been working :thinking:.
Happy to make the switch! At least on my machine there's some kind of memory leak with eslint that causes it to spin forever when trying to format. maybe only prettier will fix it!